caffeinated/admin/extend/messages/espresso_events_Messages_Hooks_Extend.class.php

```diff
@@ -198,19 +199,18 @@ public function messages_metabox($event, $callback_args)
             $tabbed_content = $tabbed_content->get_error_message();
         }
 
-        $notices = '
-	<div id="espresso-ajax-loading" class="ajax-loader-grey">
-		<span class="ee-spinner ee-spin"></span>
-		<span class="hidden">' . esc_html__('loading...', 'event_espresso') . '</span>
-	</div>
-	<div class="ee-notices"></div>';
+        $notices = '<div id="espresso-ajax-loading" class="ajax-loader-grey">
+            <span class="ee-spinner ee-spin"></span>
+            <span class="hidden">' . esc_html__('loading...', 'event_espresso') . '</span>
+        </div>
+        <div class="ee-notices"></div>';
```

Comment:
plz don't remove extra whitespace or line breaks in cases like this

Reply:
Fixed.